dists/macosx/Info.plist

```diff
@@ -4,6 +4,18 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
+	<key>CFBundleDisplayName</key>
+	<string>ScummVM</string>
+	<key>CFBundleExecutable</key>
+	<string>scummvm</string>
+	<key>CFBundleGetInfoString</key>
+	<string>1.8.0git, Copyright 2001-2016 The ScummVM Team</string>
+	<key>CFBundleIconFile</key>
+	<string>scummvm.icns</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
Looks like the `in` file hardcodes this to `org.scummvm.scummvm`. This makes this gone next time we change our source version. Always make changes to the in file and use `./devtools/update_version.pl` to re-generate the files.

Additionally: the `bundle` target in `ports.mk` has not been updated to replace the variable. This needs to be done.